### PR TITLE
Sample launch file for melodic

### PR DESCRIPTION
**Pull request by paulacvalle**
Sample launch file with all parameters listed with default values.
Also includes a quick fix to issues with remapping the camera_info topic.
I hope this might be useful for another beginner!

All the best,
Paula.

### CODE REVIEW

File: launchfile_sample.launch [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<!--
+	SAMPLE LAUNCH FILE FOR depthimage_to_laserscan for ROS melodic
```
Comment (clalancette) [inline launchfile_sample.launch]:
```suggestion
	Sample launch file for depthimage_to_laserscan
```

File: launchfile_sample.launch [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	As of September 2nd 2021.
+-->
+
+
+<launch>
+
+    <!-- Depth Image to Laser Scan Node -->
+	<node  name="depthimage_to_laserscan" pkg="depthimage_to_laserscan" type="depthimage_to_laserscan" args="">
```
Comment (clalancette) [inline launchfile_sample.launch]: The indentation (here and below) is inconsistent.  Let's use a consistent 2 spaces for indentation for all levels of the XML.

File: launchfile_sample.launch [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+		<remap from="image"			to="/camera/aligned_depth_to_color/image_raw"/> <!-- change here for your camera depth topic name. Default: "/camera/depth/image_raw" -->
+
+		<remap from="camera_info"		to="/camera/color/camera_info"/> 
+		<!-- the remap for camera_info can present issues, and by default uses the same path as the one given to the image topic. Default: "<path_to_image_topic>/camera_info>" -->
```
Comment (clalancette) [inline launchfile_sample.launch]:
```suggestion
		<!-- the remap for camera_info by default uses the same path as the one given to the image topic. Default: "<path_to_image_topic>/camera_info>" -->
```

File: launchfile_sample.launch [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+
```
Comment (clalancette) [inline launchfile_sample.launch]: Remove one of these two spaces.

File: launchfile_sample.launch [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+		<!-- PARAMETERS: -->
+		<!-- These are the default values. You can remove the lines you are not using. More information on the ROS wiki page -->
```
Comment (clalancette) [inline launchfile_sample.launch]:
```suggestion
		<!-- These are the default values. --> 
```

File: launchfile_sample.launch [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+		<param name="output_frame_id"		type="str"	value="camera_depth_frame"/> <!--default: camera_depth_frame. Frame id of the laser scan. -->
+
+	</node>
+
+
+    <!-- EXTRA: Relay node - USE THIS IF YOU NEED TO CHANGE THE PATH TO THE camera_info TOPIC AND REMAP DOESN`T WORK-->
+	<!-- 
+	<node name="relay" pkg="topic_tools" type="relay" args="/camera/color/camera_info /camera/aligned_depth_to_color/camera_info">
+	</node> 
+	-->
```
Comment (clalancette) [inline launchfile_sample.launch]: Let's just remove this.

**Review by clalancette — CHANGES_REQUESTED** [review]
Thanks for the contribution.  I've put some comments below.

Besides what I've pointed out inline, we should also put this file in a `launch` subdirectory.

**Comment by paulacvalle** [comment]
> Thanks for the contribution. I've put some comments below.
> 
> Besides what I've pointed out inline, we should also put this file in a `launch` subdirectory.

Thank you for the feedback, I have made the changes. Thank you for taking the time to reply, I am learning on how to contribute to bigger repositories from this experience!

**Review by clalancette — APPROVED** [review]
I went ahead and changed all tabs to spaces; layouts look more consistent in different editors with spaces.

Otherwise, this looks good to me.  Thanks for the contribution, I'll go ahead and merge this.

**Pull request merged**